### ISSUE TITLE
Please support automatic peering

### ISSUE THREAD

**Issue by amuttsch** [clementine-player/Android-Remote#23]
_From [devuran...@gmx.net](https://code.google.com/u/100741471971435488924/) on October 25, 2013 15:01:32_

Currently I need to enter the IP of the remote Clementine machine into the Android app. This is cumbersome and inconvenient. For example, my IP might change regularly and be different in different networks.

It would be great if Clementine Remote Android would support some more automatic form of peering, for example using:
- ZeroConf:
  Display a list of Clementine services in the network
- Bluetooth:
  1) Peer your phone with the desktop using bluetooth
  2a) Then announce some extra bluetooth service (I do not know if bluetooth allows custom services)
  2b) If that is impossible: Show the list of connected bluetooth devices and exchange connection information via file-transfer
- IPv6 multicast
  Basically a special purpose ZeroConf rip-off
- Local network broadcast
  Similar to what NetBIOS did

_Original issue: http://code.google.com/p/clementine-remote-android/issues/detail?id=23_

**Comment by nigels-com** [comment]
I'd suggest keep a history of recent IPs, but showing only those that are online.
